content/5_Plat/2DRQ.mdx

```diff
@@ -418,7 +441,7 @@ int main() {
 
 <Warning title="Implementation Note">
 
-As mentioned earlier, the above `BIT2D` implementation is significantly slower than Benq's `OffBIT2D` and, in fact, will get TLE on the Soriya's Programming Project; this is due to the large amount of calls to `vector.resize` it makes.
+As mentioned earlier, the above `OfflineBIT2D` implementation is significantly slower than Benq's `OffBIT2D` and, in fact, will get TLE on the Soriya's Programming Project.
```

Comment:
i think you can still explain why it's slower

Reply:
ok the thing is the previous explanation for why it got constant factored is wrong tho (or maybe I am underestimating the space complexity for this problem) because it doesn't call `.resize` all that many times

so I'm p sure it just gets constant factored out in this case

idk if benq's passes, but if it does it's because he does some cursed stuff and actually flattens out the binary indexed trees into just one really big binary indexed tree

Reply:
hm okay i think this is just a cursed problem because of dmoj tl updates